src/mantine-core/src/components/Spoiler/Spoiler.module.css

```diff
@@ -1,6 +1,8 @@
 .root {
   position: relative;
-  margin-bottom: rem(24px);
+  &[data-has-spoiler] {
```

Comment:
There should be a new line before this selector

Reply:
Added a new line there